### PR TITLE
Added prelim test for not missing / omitted passages

### PR DESCRIPTION
**Pull request by ghost**
E.g., Mk 16:9-20

### CODE REVIEW

**Comment by TomFaulkner** [comment]
Thanks, Gary. It looks like you've included your VS Code files, can you remove those? (I should throw in a .gitignore to prevent that going forward.)

File: tests/test_bible.py [hunk 1]
```diff
@@ -27,6 +30,11 @@ def test_includes(self):
         with self.assertRaises(RangeError):
             # acts 2 has 47 verses
             self.two_books.includes(bible.Verse('Acts 2:48'))
+        
+""" TODO: test for range of boxed/omitted passages        
+        # with self.assertRaises(RangeError):
+        #     # mark 16:9-20 has been boxed/omitted
+        #     self.boxed.includes(bible.Verse('Mark 16:9')) """
```
Comment (TomFaulkner) [inline tests/test_bible.py]: Why is this commented out and a TODO? Is the PR a work in progress?

File: tests/test_bible.py [hunk 2]
```diff
@@ -49,6 +57,8 @@ def test_smart_format(self):
         self.assertEqual(self.two_books.smart_format(),
                          'Acts 1:1 - Romans 16:27')
         print(self.two_books.smart_format())
+        self.assertEqual(self.boxed.smart_format(), 'Mark 16:9-20')
+        print(self.boxed.smart_format())
```
Comment (TomFaulkner) [inline tests/test_bible.py]: Forgot to remove the print?

Comment (ghost) [reply tests/test_bible.py]: Yes. Was there for testing

File: tests/test_bible.py (hunk 1 above)
Comment (ghost) [reply tests/test_bible.py]: Note to myself to implement RangedError exception class later on

File: tests/test_bible.py [hunk 3]
```diff
@@ -48,7 +56,7 @@ def test_smart_format(self):
         self.assertEqual(self.romans.smart_format(), 'Romans 1:1 - 16:27')
         self.assertEqual(self.two_books.smart_format(),
                          'Acts 1:1 - Romans 16:27')
-        print(self.two_books.smart_format())
```
Comment (TomFaulkner) [inline tests/test_bible.py:51]: Oops, looks like I missed a print some time ago. :open_mouth:

**Comment by ghost** [comment]
Should we treat the added test and the RangeError as two separate PRs?

**Comment by TomFaulkner** [comment]
That is fine, but right now there is commented out code.

File: tests/test_bible.py [hunk 4]
```diff
@@ -28,6 +31,11 @@ def test_includes(self):
             # acts 2 has 47 verses
             self.two_books.includes(bible.Verse('Acts 2:48'))
 
+        """ TODO: test for range of boxed/omitted passages        
+        with self.assertRaises(RangeError):
+            # mark 16:9-20 has been boxed/omitted
+            self.boxed.includes(bible.Verse('Mark 16:9')) """
+
```
Comment (TomFaulkner) [inline tests/test_bible.py]: Why is this commented out?

Comment (ghost) [reply tests/test_bible.py]: Removed lines for now. It's a TODO for now. Will work on RangeError exception class soon

**Pull request closed without merging**